requirements.txt

```diff
@@ -0,0 +1 @@
+datamodel-code-generator
```

Comment:
This shouldn't be necessary, right? This package was only necessary to initially transform json-schema to Pydantic, right?